### ISSUE TITLE
424107D1 - Zegapain XOR Demo

### ISSUE THREAD

**Issue by Nezarn** [xenia-project/game-compatibility#100]
Tested with https://github.com/benvanik/xenia/commit/baa86fcd1b2c3d516faedacb63c83147ee244a84

Instant crash. (state-nothing label)

[Marketplace](http://marketplace.xbox.com/ja-JP/Product/%E3%82%BC%E3%83%BC%E3%82%AC%E3%83%9A%E3%82%A4%E3%83%B3-XOR-%E4%BD%93%E9%A8%93%E7%89%88/00000000-0000-400d-80d0-0001424107d1)

[log](https://github.com/Nezarn/Xenia-Logs/blob/master/424107D1%20-%20Zegapain%20XOR%20Demo%20log%20(baa86fc).txt)

**Comment by DrChat** [comment]
Looks like it's crashing in xenia - have you tried running it more than once?

**Comment by Nezarn** [comment]
Yep, log is always the same (except sometimes memory dump is missing from log)

![crash](https://cloud.githubusercontent.com/assets/3817985/8771926/2f8b582e-2ec7-11e5-9be0-1f742100be86.PNG)

and here only module name(?) changes, sometimes its xenia.exe, sometimes its ntdll.dll

**Comment by bdeimos** [comment]
Tested on (https://github.com/xenia-project/xenia/commit/f89b4626a37dd4d1e5a817909252d3d4dd054299)

3D not drawn.

(state-gameplay gpu-missing-drawing)

[xenia.zip](https://github.com/xenia-project/game-compatibility/files/2151763/xenia.zip)

![untitled](https://user-images.githubusercontent.com/40533288/42125847-11d210dc-7c87-11e8-9819-802306f2716b.jpg)
